database/migrations/2020_05_22_020303_alter_metric_column.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AlterMetricColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('application_metrics', function (Blueprint $table) {
+            $table->string('metric', 64)->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('application_metrics', function (Blueprint $table) {
+            $table->string('metric', 32)->change();
```

Comment:
I dont know this of the top of my head but wouldn't this cause issues if an entry exceeds the 32c limit by the time you revert the migration?

Reply:
it will be cutted down to 32 characters